Sources/Sentry/include/SentryOutOfMemoryTracker.h

```diff
@@ -0,0 +1,28 @@
+#import "SentryDefines.h"
+
+@class SentryOptions, SentryOutOfMemoryLogic, SentryDispatchQueueWrapper;
+
+NS_ASSUME_NONNULL_BEGIN
+
+static NSString *const SentryOutOfMemoryExceptionType = @"Out Of Memory";
```

Comment:
Turns out the naming here should be snake case:

`out_of_memory` instead.

Take it up with @mitsuhiko on this

Reply:
Thanks @bruno-garcia 💯